be/src/storage/compaction_utils.cpp

```diff
@@ -117,8 +116,8 @@ CompactionAlgorithm CompactionUtils::choose_compaction_algorithm(size_t num_colu
 }
 
 std::unique_ptr<CompactionPolicy> CompactionUtils::create_compaction_policy(CompactionContext* context) {
-    // now only support LevelCompactionPolicy
```

Comment:
Original is for what?

Reply:
the compaction policy is the same as original compaction strategy. So I use original.

Reply:
change it to base_cumulative